platform/Cargo.toml

```diff
@@ -8,6 +8,7 @@ documentation.workspace = true
 
 [dependencies]
 async-trait = "0.1.63"
+aws-sdk-secretsmanager = "0.24.0"
```

Comment:
I think we will want to go a different way based on feedback from @talentedmrjones in yesterday's gloaming.  You can get with him to get the details.